lib/cosmos/tools/script_runner/script_runner.rb

```diff
@@ -164,6 +164,16 @@ def initialize_actions
       @edit_comment.statusTip = tr('Comment/Uncomment Lines')
       @edit_comment.connect(SIGNAL('triggered()')) { active_script_runner_frame().comment_or_uncomment_lines }
 
+      @edit_zoom_in = Qt::Action.new(tr('Increase Font Size'), self)
+      @edit_zoom_in_keyseq = Qt::KeySequence.new(Qt::KeySequence::ZoomIn)
+      @edit_zoom_in.shortcut  = @edit_zoom_in_keyseq
+      @edit_zoom_in.connect(SIGNAL('triggered()')) { active_script_runner_frame().zoom_in }
```

Comment:
Can we make these work with Ctrl + scroll wheel

Reply:
I map to the default ZoomIn and ZoomOut key seqeuences of the platform. You want something more special than that? :-)

Reply:
Yes - I want the scroll wheel :)